ubuntu/Dockerfile

```diff
@@ -9,9 +12,13 @@ RUN apt-get update -qq \
  && rm -rf /packages /var/lib/apt/lists/*
 
 RUN cd /tmp \
- && git clone https://github.com/root-project/root /usr/src/root \
+ && git clone ${ROOT_PROJECT} /usr/src/root \
```

Comment:
No need to use git for building from a release, we can just download the tarball instead, which is much smaller.

Reply:
Yes, good idea.

Done in https://github.com/root-project/root-docker/pull/11/commits/5e6a239a90f767e51eca118331d2007a43b4759e